### PR TITLE
Front/ar 25/en tant qu utilisateur je souhaite pouvoir voir les actions r actions crees par l utilisateur

### PR DESCRIPTION
**Pull request by Vivelis**
* **Checklist before requesting a review**
- [x] The commits messages follows our guidelines
- [x] I have performed a self-review of my code
- [x] I have documented my code
- [ ] Do we need to implement unit tests?

* **In what state is your pull request?**
- [x] Ready to merge / Waiting a reviwer to see my work.
- [ ] Work In Progress (WIP) / My work is not finish but i want daily reviews. (Draft)
- [ ] CI Review / I want to see what the CI thinks of my work.
- [ ] Need feedback / Just to have feedback on what i produced. (May not be merged)

* **What kind of changes does this PR introduce?** (You can choose multiple)
- [ ] Bug fix
- [x] Feature request
- [ ] New / Updated documentation
- [ ] Testing CI ( Make the pull request in draft mode)

* **Describe your changes**

En tant qu'utilisateur, je souhaite pouvoir voir les actions-réactions créées par l'utilisateur

* **Issue ticket number and link**

[AR-25](https://bravmm.atlassian.net/browse/AR-25)

[AR-25]: https://bravmm.atlassian.net/browse/AR-25?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

## Summary by CodeRabbit

- **New Features**
  - Introduced an area status indicator with color-coded pastilles to visually represent the active or inactive status of areas.

- **Enhancements**
  - Improved the AreaIcon component to include dynamic rendering based on the area's status.

- **Documentation**
  - Added an AreaDetailsInterface to standardize the structure of area data across the application.

- **Refactor**
  - Updated the Services component to fetch and display area details using a new `getAreas` function.

### CODE REVIEW

File: front/app/Components/AreaIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Image from "next/image"; // Import de next/image
+
+interface AreaIconProps {
```
Comment (romainpanno) [inline front/app/Components/AreaIcon.tsx:4]: Create an interface file for that in the folder interfaces

**Review by romainpanno — CHANGES_REQUESTED** [review]

File: front/app/Components/AreaIcon.tsx (hunk 1 above)
Comment (Vivelis) [reply front/app/Components/AreaIcon.tsx:4]: Je le fait dans quel dossier ?

File: front/app/services/page.tsx [hunk 2]
```diff
@@ -1,10 +1,81 @@
...
+      status: true,
+    },
+    {
+      name: "Facebook",
+      image: "/logo.svg",
+      status: true,
+    },
+  ];
+
+  return _areas;
```
[the remark this answers is not in the document]
Comment (Vivelis) [reply front/app/services/page.tsx]: It's ok, I will replace it in a new pull request

**Review by romainpanno — APPROVED** [review]

**Review by Beafowl-Pull — APPROVED** [review]
RAS (attention au /logo.svg)

**Pull request merged**